enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotQuestion.tsx

```diff
@@ -0,0 +1,41 @@
+import { useMemo, useState } from "react";
+import { t } from "ttag";
+
+import { InteractiveAdHocQuestion } from "embedding-sdk/components/private/InteractiveAdHocQuestion";
+import { withPublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
+import { Flex, Text } from "metabase/ui";
+
+import { MetabotChatEmbedding } from "./MetabotChatEmbedding";
+
+const MetabotQuestionInner = () => {
+  const [result, setResult] = useState<Record<string, any> | null>(null);
+
+  const redirectUrl = useMemo(() => {
+    return result?.payload?.payload?.data?.reactions?.find(
```

Comment:
In the future I feel `onResult` should have the responsibility of returning the formatted reactions for us, seems a bit weird to have to parse this in the SDK component

Reply:
oh lol, glad to see hackathon projects coming to life!

Reply:
Fixed with 1aad450ee3cab3bcacf3e9b58886384b696ad6ce